gateway-spi/src/main/java/org/apache/knox/gateway/SpiGatewayMessages.java

```diff
@@ -91,10 +91,10 @@ public interface SpiGatewayMessages {
             text = "The dispatch to {0} was disallowed because it fails the dispatch whitelist validation. See documentation for dispatch whitelisting." )
   void dispatchDisallowed(String uri);
 
-  @Message( level = MessageLevel.DEBUG, text = "HTTP client connection timeout is set to {0} for {1}" )
+  @Message( level = MessageLevel.INFO, text = "HTTP client connection timeout is set to {0} msec for {1}" )
   void setHttpClientConnectionTimeout(int connectionTimeout, String serviceRole);
 
-  @Message( level = MessageLevel.DEBUG, text = "HTTP client socket timeout is set to {0} for {1}" )
+  @Message( level = MessageLevel.INFO, text = "HTTP client socket timeout is set to {0} msec for {1}" )
   void setHttpClientSocketTimeout(int csocketTimeout, String serviceRole);
```

Comment:
nit: typo in `csocketTimeout`: either `clientSocketTimeout` or `socketTimeout` (I prefer the 2nd one).